doc/source/api/core/solver/events.rst

```diff
@@ -4,16 +4,16 @@ EventsManager
 =============
 
 An instance of ``EventsManager`` exists as an active ``events_manager`` property in each
-solution-mode session object. It allows to register client callback with server side events.
-Whenever event occurs, the registered callback is triggered with session id and event info 
-passed as arguments to callback. It is useful in solution monitoring and updating graphics 
-during run time.
+solution-mode session object. It allows for registering the client callback with server-side
+events. Whenever an event occurs, the registered callback is triggered with session ID and event
```

Comment:
So, actually, here the occurrence of events is mentioned. So we can simplify the above, and rewrite the whole thing.